FAQ.md

```diff
@@ -1,5 +1,23 @@
 # FAQ
 
+<!-- Table of contents generated with DocToc: https://github.com/thlorenz/doctoc -->
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+
+- [If nobody watches a video, is it seeded?](#if-nobody-watches-a-video-is-it-seeded)
+- [What is WebSeed?](#what-is-webseed)
+- [If a client requests each chunk of a video through HTTP, will the server be overloaded?](#if-a-client-requests-each-chunk-of-a-video-through-http-will-the-server-be-overloaded)
+- [Will an index of all the videos of servers you follow be too large for small servers?](#will-an-index-of-all-the-videos-of-servers-you-follow-be-too-large-for-small-servers)
+- [What codecs can I use for the videos I want to upload?](#what-codecs-can-i-use-for-the-videos-i-want-to-upload)
+- [I want to change my host, how can I do that?](#i-want-to-change-my-host-how-can-i-do-that)
+- [Should I have a big server to run PeerTube?](#should-i-have-a-big-server-to-run-peertube)
+- [Are you gonna use the Steem blockchain?](#are-you-gonna-use-the-steem-blockchain)
+- [Are you gonna support adding ads?](#are-you-gonna-support-adding-ads)
```

Comment:
nit: `gonna` is familiar, please use `going to` instead (as well in anchors)

Reply:
Also I'd rephrase the later to `are you going to support advertisements?`

Reply:
I wasn't sure about the "support advertisments" but now that I think of it, that's how everyone phrases it.